pom.xml

```diff
@@ -50,6 +50,12 @@
 			<scope>test</scope>
 		</dependency>
 
+      <dependency>
+        <groupId>org.mockito</groupId>
+        <artifactId>mockito-core</artifactId>
+        <version>1.10.19</version>
+        <scope>test</scope>
+      </dependency>
 
```

Comment:
I am not seeing this library being used anywhere in this pull request? is this needed?